mne/chpi.py

```diff
@@ -0,0 +1,437 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from os import path as op
+from scipy import linalg
+
+from .io.pick import pick_types
+from .io.base import _BaseRaw
+from .io.constants import FIFF
+from .forward import (_magnetic_dipole_field_vec, _create_meg_coils,
+                      _concatenate_coils)
+from .cov import make_ad_hoc_cov, _get_whitener_data
+from .transforms import apply_trans, invert_transform
+from .utils import verbose, logger
+from .fixes import partial
+from .externals.six import string_types
+
+
+# ############################################################################
+# Reading from text or FIF file
+
+@verbose
+def get_chpi_positions(raw, t_step=None, verbose=None):
+    """Extract head positions
+
+    Note that the raw instance must have CHPI channels recorded.
+
+    Parameters
+    ----------
+    raw : instance of Raw | str
+        Raw instance to extract the head positions from. Can also be a
+        path to a Maxfilter log file (str).
+    t_step : float | None
+        Sampling interval to use when converting data. If None, it will
+        be automatically determined. By default, a sampling interval of
+        1 second is used if processing a raw data. If processing a
+        Maxfilter log file, this must be None because the log file
+        itself will determine the sampling interval.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    translation : array
```

Comment:
I would put the shape in the first line

translation : array, shape (n_times, 3)